website/content/docs/ecs/terraform/install.mdx

```diff
@@ -80,10 +88,226 @@ The following fields are required. Refer to the [module reference documentation]
+| `load_balancer_target_group_arn` | string | Specifies the ARN of an existing load balancer target group. The load balancer target group allows ingress to the gateway task. <br/>No additional load balancer configuration is necessary. Only NLBs and ALBs are supported. The container name and port are set automatically based on other fields. |
+
+#### Mesh init
+
+The `mesh-init` container is a short-lived container that sets up the initial configurations for Consul and Envoy. The `gateway-task` module automatically configures the `mesh-init` container based on the inputs specified in the [task definition](#task-definition) and [ECS service](#ecs-service) configuration.
+
+For additional information, refer to [Task Startup](/docs/ecs/architecture#task-startup) for additional information.
+
+#### Gateway task configuration examples
+
+The following examples illustrate how to configure the `gateway-task` for different use cases.
+
+##### Ingress
+
+Mesh gateways need to be reachable over the WAN to route traffic between datacenters. Configure the following options in the `gateway-task` to enable ingress through the mesh gateway.
+
+| Option | Type | Description |
+| ---    | ---  | ---         |
+| `lb_enabled` | Boolean | Set to `true` to automatically deploy and configure a network load balancer for ingress to the mesh gateway. |
+| `lb_vpc_id` | string | Specifies the VPC to launch the load balancer in. |
+| `lb_subnets` | list of strings | Specifies one or more public subnets to associate with the load balancer. |
+
+<CodeBlockConfig filename="mesh-gateway.tf">
+
+```hcl
+module "my_mesh_gateway" {
+  ...
+
+  lb_enabled = true
+  lb_vpc_id  = "<VPC ID>"
+  lb_subnets = ["<public subnet IDs>"]
+}
+```
+
+</CodeBlockConfig>
 
-### Running Terraform
+Alternatively, you can manually configure ingress to the mesh gateway and provide the `wan_address` and `wan_port` inputs to the `gateway-task` module. The `wan_port` field is optional. Port `8443` is used by default.
+
+<CodeBlockConfig filename="mesh-gateway.tf">
+
+```hcl
+module "my_mesh_gateway" {
+  ...
+
+  wan_address = "<public WAN address>"
+  wan_port    = <public WAN port>
+}
+```
+
+</CodeBlockConfig>
+
+Mesh gateways route L4 TCP connections and do not terminate mTLS sessions. If you manually configure [AWS Elastic Load Balancing](https://aws.amazon.com/elasticloadbalancing/) for ingress to a mesh gateway, you must use an AWS [Network Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/network/introduction.html) or a [Classic Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/classic/introduction.html).
+
+
+#### ACLs
+
+Configure the following options in the `gateway-task` when ACLs are enabled.
+
+| Option | Type | Description |
+| ---    | ---  | ---         |
+| `acl` | Boolean | Set to `true` if ACLs are enabled. |
```

Comment:
> I'm not sure why `Boolean` is capitalized, maybe this is intentional? It is not capitalized in the first table and none of the other type names are capitalized (`string`). It is also not capitalized in the other files that it appears in.
> 
> The actual Terraform type for this is `bool`. I don't know if we need to be that strict about the type since it is already in use as `boolean` throughout these ECS docs.

@cthain My understanding is that Boolean is the only data type with a proper name and should be capitalized. I believe it's on par with descriptions like Jacobean or Victorian. The lowercase form is actually the incorrect usage.

Reply:
I don't have any strong feelings about the capitalization but I think we should try to be consistent in our usage throughout the docs. There is a mix of `Boolean`, `boolean` and `bool` throughout the docs and I know this predates these updates so I'm fine with whatever you decide to do here.